src/Test.FunctionalTests.Sarif/Multitool/ValidateCommandTests.cs

```diff
@@ -158,6 +166,11 @@ protected override string ConstructTestOutputFromInputResource(string inputResou
             // We can't transform these test inputs as that operation fixes up erros in the file.
             bool updateInputsToCurrentSarif = ruleUnderTest.StartsWith("SARIF") ? true : false;
```

Comment:
@lgolding , based on this rule, I created a new parameter which the default is false to override it. What do you think?